[32mINFO    [0m juju.model:model.py:2972 Waiting for model:
  mongos-k8s/0 [idle] active:
[32mINFO    [0m pytest_operator.plugin:plugin.py:834 Model status:

Model  Controller          Cloud/Region        Version  SLA          Timestamp
test   microk8s-localhost  microk8s/localhost  3.5.3    unsupported  01:38:26Z

App            Version  Status  Scale  Charm        Channel  Rev  Address         Exposed  Message
application             active      1  application             0                  no       
config-server           active      1  mongodb-k8s  6/edge    56  10.152.183.52   no       
mongos-k8s              active      1  mongos-k8s              0  10.152.183.159  no       
shard0                  active      1  mongodb-k8s  6/edge    56  10.152.183.136  no       Primary

Unit              Workload  Agent  Address       Ports  Message
application/0*    active    idle                        
config-server/0*  active    idle   10.1.131.207         
mongos-k8s/0*     active    idle   10.1.131.202         
shard0/0*         active    idle   10.1.131.208         Primary

[32mINFO    [0m pytest_operator.plugin:plugin.py:840 Juju error logs:

unit-config-server-0: 01:32:14 ERROR unit.config-server/0.juju-log config-server:4: Failed to add shard shard0 to the config server, error=OperationFailure("Authentication failed., full error: {'ok': 0.0, 'errmsg': 'Authentication failed.', 'code': 18, 'codeName': 'AuthenticationFailed', '$clusterTime': {'clusterTime': Timestamp(1728869533, 3), 'signature': {'hash': b'S}\\x1e\\xaa\\x82$\\x0e:3h\\x1e\\x8b\\xde\\x9f\\xb03e\\x88}\\xbe', 'keyId': 7425437888537427993}}, 'operationTime': Timestamp(1728869533, 3)}")
unit-config-server-0: 01:32:14 ERROR unit.config-server/0.juju-log config-server:4: shard0 shard does not have the same auth as the config server.

[32mINFO    [0m pytest_operator.plugin:plugin.py:904 Forgetting main...